src/AutoBuild.jl

```diff
@@ -1416,6 +1416,124 @@ function build_jll_package(src_name::String,
         !ispath(pkg_dir) && mkdir(pkg_dir)
         write(joinpath(pkg_dir, "platform_augmentation.jl"), augment_platform_block)
 
+        overload_parse = raw"""
+            # Update Base.parse to support riscv64, needed for Julia <1.12
+            @static if !haskey(BinaryPlatforms.arch_mapping, "riscv64")
```

Comment:
Good idea checking this, we can already parse riscv64 triples already in v1.10.8 and v1.11.3 :+1: